posthog/models/event/util.py

```diff
@@ -79,6 +79,7 @@ def create_event(
         "group2_created_at": format_clickhouse_timestamp(group2_created_at, ZERO_DATE),
         "group3_created_at": format_clickhouse_timestamp(group3_created_at, ZERO_DATE),
         "group4_created_at": format_clickhouse_timestamp(group4_created_at, ZERO_DATE),
+        "person_mode": "full",
```

Comment:
This and the bulk insert below look like they are for demo/tests. I could have just left them out and let the CH default take over, but I'd rather be explicit.